### PR TITLE
antool: two minor fixes

### PR DESCRIPTION
**Pull request by ldorau**
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmemfile/417)

### CODE REVIEW

**Comment by krzycz** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/417)*

File: tests/antool/test-fi.sh [hunk 1]
```diff
@@ -39,8 +39,8 @@ function cut_out_from_file() { # file offset1 offset2
 	OFFSET_1=$2
 	OFFSET_2=$3
 	OUTPUT=$(mktemp)
-	dd if=$FILE of=$OUTPUT  bs=$OFFSET_1 count=1 status=noxfer 2>/dev/null
-	dd if=$FILE of=$OUTPUT ibs=$OFFSET_2 skip=1 obs=$OFFSET_1 seek=1 status=noxfer 2>/dev/null
+	dd if=$FILE count=1 bs=$OFFSET_1                      status=noxfer 2>/dev/null >  $OUTPUT
+	dd if=$FILE seek=1 ibs=$OFFSET_2 skip=1 obs=$OFFSET_1 status=noxfer 2>/dev/null >> $OUTPUT
```
Comment (marcinslusarz) [inline tests/antool/test-fi.sh]: This doesn't seem right. When $OFFSET_1 != 0 I think it will have different result.

Comment (ldorau) [reply tests/antool/test-fi.sh]: It works very well, $OFFSET_1 is always != 0

**Comment by krzycz** [comment]
Reviewed 1 of 2 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/417)*

**Comment by marcinslusarz** [comment]
:lgtm:

---

Reviewed 1 of 2 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/417#-:-Kw4U4kU7IxRSU8bsgFc:bnfp4nl)*

**Comment by ldorau** [comment]
@marcinslusarz 
```
The following tests FAILED:
	 36 - mt_rename_drd (Failed)
Errors while running CTest
```

**Comment by marcinslusarz** [comment]
The issue in mt_rename_drd is fixed by #422.

**Pull request merged**